### PR TITLE
Feature/activity details

### PR DESCRIPTION
**Pull request by thaliyil**
plz check:)

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import GlobalStyle from "../styles";
+//import activities from "@/assets/activities";
```
Comment (MerleMak) [inline pages/_app.js:2]:
```suggestion
```

**Review by MerleMak — COMMENTED** [review]
This looks great!
Only one thing: I think your fomatter (prettier) is not active. Please confirm in your VS Code Settings that the checkbox "format on save" is  checked and the your "Defaut Formatter" is set to "Prettier"

Nice job! Keep up the good work!

**Comment by MerleMak** [comment]
It seems like your deployment is failing though, are you looking at that? :)

File: components/ActivityCardPreview/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  border: 1px solid grey;
+  padding: 5px;
+  border-radius: 5px;
+  margin: 10px 0px;
+`;
+
+const StyledImage = styled(Image)`
+  width: auto;
+  height: auto;
+`;
```
Comment (EmmaDir) [inline components/ActivityCardPreview/index.js:23]:
```suggestion

```

Comment (EmmaDir) [reply components/ActivityCardPreview/index.js:23]: Not necessary, you already defined height and width in line 34, 35

File: components/ActivityCardPreview/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+export default function ActivityCardPreview({ activity }) {
+  const { title, categoryIds, imageUrl } = activity;
+  const filterCategories = getFilteredCategories(categoryIds);
+  return (
+    <>
+      <h2>{title}</h2>
```
Comment (EmmaDir) [inline components/ActivityCardPreview/index.js:30]: Make sure to have a h1 before using the h2
```suggestion
      <h1>{title}</h1>
```

File: pages/_app.js (hunk 1 above)
Comment (EmmaDir) [inline pages/_app.js:2]: Remove comments before merging
```suggestion
```

File: pages/index.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
+import ActivitiesList from "@/components/ActivitiesList";
 export default function HomePage() {
   return (
     <div>
```
Comment (EmmaDir) [inline pages/index.js:4]:
```suggestion
    <section>
```

File: pages/index.js [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
-      <h1>Hello from Next.js</h1>
+      <h1>TrailBlazer</h1>
+      <ActivitiesList />
     </div>
```
Comment (EmmaDir) [inline pages/index.js:7]: Avoid using div
```suggestion
    </section>
```

**Review by EmmaDir — COMMENTED** [review]
Looks good! Just a few small suggestions for improvement😊

File: components/ActivityCardPreview/index.js (hunk 3 above)
Comment (RebeccaPavelt) [reply components/ActivityCardPreview/index.js:30]: we already have an h1 in the parent element, so when rendering there would be two h1s.

**Pull request merged**